example/HTTP/latency_v2.lua

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env tarantool
+package.path = package.path .. ";../?.lua"
+
+local json = require('json')
+local fiber = require('fiber')
+local metrics = require('metrics')
+local http_middleware = metrics.http_middleware
+
+-- Configure HTTP routing
+local ip = '127.0.0.1'
+local port = 12345
+local httpd = require('http.server').new(ip, port) -- HTTP ver. 2.x.x
+local router = require('http.router').new() -- HTTP ver. 2.x.x
+local route = { path = '/path', method = 'POST' }
+
+-- Set route handler
+local handler = function(req)
+    for _ = 1, 10 do
+        fiber.sleep(0.1)
+    end
+
+    return { status = 200, body = req.body }
+end
+router:route(route, handler)
+
+-- Configure average latency collector
+local collector = http_middleware.build_default_collector(
+    'average', 'path_latency',
+    'My collector for /path requests latency'
+)
+
+-- Set router average latency collection middleware
+router:use(
+    http_middleware.v2(collector),
+    { name = 'latency_instrumentation', path = route.path, method = route.method }
```

Comment:
Let's remove path and method restriction, or change it to show that middleware is applied to multiple routes.

I think most of users would enable metrics for all routes, because it's simple.

Reply:
Removed path and method middleware tags